sympy/stats/crv_types.py

```diff
@@ -159,6 +159,9 @@ def ContinuousRV(symbol, density, set=Interval(-oo, oo)):
     """
     Create a Continuous Random Variable given the following:
 
+    Many common continuous random variable types are already implemented.
```

Comment:
The sentence above with the colon should probably be changed as well.

Reply:
I should clarify, because the diff is rather misleading. The second sentence wasn't actually changed, other than its location. In fact, most of the apparent additions in this PR (thus far) are really just moves -- the result of reordering certain docstring sections to comply with [documentation style guide](https://docs.sympy.org/latest/documentation-style-guide.html#docstring-sections). For example, in this case, [Parameters and Returns](https://github.com/sympy/sympy/blob/089bef23e02b311bd8a5b4400ccd8a5d427005d5/sympy/stats/crv_types.py#L162-L175) were moved below [Examples](https://github.com/sympy/sympy/blob/089bef23e02b311bd8a5b4400ccd8a5d427005d5/sympy/stats/crv_types.py#L181-L195).

Would it be best to merge those moves quickly and pick up the rest of this effort in a new PR? I imagine it would make things much easier to review!

In the meantime, I tightened up the summary line (first sentence) a bit. Both it and the short summary could use a revisit at some point.